File: Soluções/1010.js
var input = require('fs').readFileSync('./Index.txt', 'utf8');
var lines = input.split('\n');

const pegarValores = (linha) => linha.split(' ').map((item) => Number(item));

let [num1, num2, num3] = pegarValores(lines.shift())
let [num4, num5, num6] = pegarValores(lines.shift())

Number(num2, num3, num5, num6)

let valor = num2 * num3 + num5 * num6 

console.log(`VALOR A PAGAR: R$ ${valor.toFixed(2)}`);

File: C/1010.c
#include <stdio.h>

int main(){
    int a, b, c, d;
    float A, B, calc;
    
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%f", &A);
    
    scanf("%d", &c);
    scanf("%d", &d);
    scanf("%f", &B);
    
    calc = (b * A + d * B);
    
    printf("VALOR A PAGAR: R$ %.2f\n", calc);

    return 0;
}